doc/host_config_schema/cchost_config.json

```diff
@@ -76,9 +76,14 @@
                 "properties": {
                   "authority": {
                     "type": "string",
-                    "enum": ["Node", "Service"],
+                    "enum": ["Node", "Service", "ACME", "NONE"],
                     "default": "Service",
-                    "description": "The type of endorsement for the TLS certificate used in client sessions. If the endorsement is not available, client sessions will be terminated, before the TLS handshake is complete. 'Node' means self-signed, 'Service' means service-endorsed."
+                    "description": "The type of endorsement for the TLS certificate used in client sessions. If the endorsement is not available, client sessions will be terminated, before the TLS handshake is complete. 'Node' means self-signed, 'Service' means service-endorsed, 'ACME' means an ACME-capable CA, 'NONE' means no authority (for http)."
```

Comment:
I think the `NONE` needs to be worded much strongly as it removes confidentiality for client requests.

Reply:
Good point, will do.